etcd-manager/dev/add-symlink.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash -e
```

Comment:
`-e` will cause the script to exit on error. Why is this flag used here, but not in the other scripts?

Reply:
Looks -e was here since the initial version, and i just want to fix the interpreter issue first in this pr